environment.devenv.yml

```diff
@@ -1,26 +1,28 @@
 #
 # Conda devenv environment for paquo
 #
-# Create the environment, do not install pado in development mode:
+# Create the environment:
 #   conda devenv
-# equivalent to:
-#   PAQUO_DEVEL="" conda devenv
-# Create the environment, install pado in development mode:
-#   PAQUO_DEVEL=TRUE conda devenv
 #
 
-{% set PAQUO_DEVEL = os.environ.get('PAQUO_DEVEL', False) %}
-{% set WITH_OME = os.environ.get('WITH_OME', False) %}
-
 name: paquo
 
 channels:
   - conda-forge
+  - defaults
```

Comment:
Please remove the `defaults` channel here. It potentially requires licenses. Better to leave that up to the user they have anaconda installation with activated licenses and want to have defaults as the base channel.